sea-query-binder/src/sqlx_sqlite.rs

```diff
@@ -73,6 +73,8 @@ impl<'q> sqlx::IntoArguments<'q, sqlx::sqlite::Sqlite> for SqlxValues {
                 Value::ChronoDateTimeWithTimeZone(t) => {
                     args.add(t.map(|t| *t));
                 }
+                #[cfg(feature = "postgres-interval")]
+                Value::Interval(_) => {}
```

Comment:
Is there a better of doing this other than splitting the query builder or throwing an error?